test/src/org/labkey/test/tests/cds/CDSGroupTest.java

```diff
@@ -488,6 +493,13 @@ private void verifyLinksOnStudyPage(String descr)
         verifyCuratedLink(descr);
     }
 
+    private void verifyExportButtonOnReports()
+    {
+        cds.viewLearnAboutPage("Reports");
```

Comment:
You might have to add a report, otherwise Reports tab will not be present if there are no reports; unless a test that creates a report always ran before this test.

Reply:
VerifyExportButtonOnReports is added at the end of verifyInteractiveAndCuratedLinks() test, as per the test plan that test adds reports.
Am i missing something?

Reply:
Nope, I missed that.

Looks good!